src/github-client.js

```diff
@@ -10,7 +10,7 @@ import jsonqueryEngine from 'jsonquery-engine';
 
 
 
-import Octo from 'octokat/dist/node/base';
+import Octo from 'octokat';
```

Comment:
I wasn't sure, I was trying to fix the issue in the octokat. This change might not have been necessary.